FIX: Change the link and format of cheatsheet to keep consistency. (#40)
ansys/pyansys-cheat-sheet: README.rst

               :link: https://cheatsheets.docs.pyansys.com/<your_tex_file_name>.pdf

Cheat sheets
~~~~~~~~~~~~~

> In the `Documentation <https://cheatsheets.docs.pyansys.com/>`_ page, you can find a comprehensive collection of cheat sheets specifically 

Author: ```suggestion
In the `Documentation <https://cheatsheets.docs.pyansys.com/>`_ page, you can find a comprehensive collection of cheatsheets specifically 
```

> In the `Documentation <https://cheatsheets.docs.pyansys.com/>`_ page, you can find a comprehensive collection of cheat sheets specifically 

Author: Done.

> designed for various PyAnsys products. These cheat sheets are carefully crafted to provide concise and practical information, serving as 

Author: ```suggestion
designed for various PyAnsys products. These cheatsheets are carefully crafted to provide concise and practical information, serving as 
```

valuable references for your development or learning journey with PyAnsys. Feel free to post issues and other questions 

> at `PyAnsys cheat sheet issues <https://github.com/ansys/pyansys-cheat-sheet/issues>`_. This is the best place to post questions and code.

Author: ```suggestion
at `PyAnsys cheatsheet issues <https://github.com/ansys/pyansys-cheat-sheet/issues>`_. This is the best place to post questions and code.
```
